controllers/controllers/cluster_controller.go

```diff
@@ -116,5 +126,30 @@ func (r *ClusterReconciler) reconcile(ctx context.Context, cluster *anywherev1.C
 }
 
 func (r *ClusterReconciler) reconcileDelete(ctx context.Context, cluster *anywherev1.Cluster, log logr.Logger) (ctrl.Result, error) {
+	capiCluster := &clusterv1.Cluster{}
+	capiClusterName := types.NamespacedName{Namespace: "eksa-system", Name: cluster.Name}
+	r.log.Info("Deleting", "name", cluster.Name)
```

Comment:
Should we be using the `r.log` logger or the logger we've been explicitly instructed to use as a parameter?

Reply:
r.log. Parameter removed

Reply:
should we include the kind in log msg too? ("kind", CAPIClusterKind, "name", cluster.Name)

Reply:
> just to understand a bit about the delete logic here based on the code: we fetch CAPI cluster obj and delete it first. If succeeded, we requeue. Since CAPI cluster is deleted, the method falls into the `apierrors.IsNotFound(err)` and we then clean up the eks-a objects.
> 
> is this right?

yes